index.html

```diff
@@ -260,23 +260,8 @@
         class="container flex flex-wrap justify-start items-center p-5"
         data-cy="sidebar"
       >
-        <button class="sideBarBtn">
-          <span
-            data-icon="hamburgerMenu"
-            data-size="20"
-            data-class="text-ys-backgroundAndText dark:text-ys-textAndIconsLight"
-          ></span>
-        </button>
         <div class="flex items-center gap-4 pr-5">
-          <p class="text-sm uppercase">Project name</p>
-          <input type="text" class="hidden" id="projectName" />
-          <button class="sideBarBtn w-auto">
-            <span
-              data-icon="editPencil"
-              data-size="12"
-              data-class="dark:text-ys-textAndIconsLight"
-            ></span>
-          </button>
+          <p id="proName" class="text-sm uppercase">Project name</p>
```

Comment:
Consider the accessibility of the p tag,

how about you try something like

```suggestion
          <p id="proName" contentEditable="true" tabindex="0" role="textbox" aria-label="Editable Project Name" class="text-sm uppercase">Project name</p>
```